openeo_processes_dask/process_implementations/udf/udf.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+
+import dask.array as da
+import xarray as xr
+from openeo.udf import UdfData
+from openeo.udf.run_code import run_udf_code
+from openeo.udf.xarraydatacube import XarrayDataCube
+
+from openeo_processes_dask.process_implementations.data_model import RasterCube
+
+__all__ = ["run_udf"]
+
+
+def run_udf(
+    data: da.Array, udf: str, runtime: str, context: Optional[dict] = None
+) -> RasterCube:
+    data = XarrayDataCube(xr.DataArray(data))
```

Comment:
Shouldn't there also be some kind of chunking here (or at least some kind of option to enable it), to make sure it can be aligned a bit with backends that automatically apply chunking (like geopyspark driver).

(I don't know the dask driver details very well, so chunking might already be in play at an other level)

Reply:
Indeed, but as far as I understood @jdries and @VincentVerelst wanted to have a first implementation ready soon and later on focus on the chunking part.

Reply:
Correct, I did now try it locally, and it seems that it also works as a callback to an apply. 
I do see that chunking is not necessarily present, but think that is caused by having a very small input array to begin with. I would propose to continue with wrapping up this one and then indeed add the more advanced tests.